smallweb.txt

```diff
@@ -73,7 +73,6 @@ http://postabdn.com/feed/
 http://prithvirajva.com/feed.xml
 http://radar.spacebar.org/f/a/weblog/rss/1
 http://redino.net/blog/feed/
-http://research.swtch.com/feed.atom
```

Comment:
I kept the https link for this site, which is sorted below rather than next to this link